tests/ops/fixtures/saas/braze_fixtures.py

```diff
@@ -171,6 +172,8 @@ def braze_erasure_data(
 
     assert response.ok
 
+    sleep(30)
```

Comment:
so i guess we don't just increase retries and/or timeout on `poll_for_existence` here because it is giving us some false positives, i.e. the data is _not totally_ there on the system/we get inconsistent results?

if so, then i guess this'll be fine as a patch...but maybe we can look to improve `poll_for_existence` to require a certain number of successful retrievals? do you think that'd be effective? doesn't necessarily need to be in this PR if you think it'd be complex.

either way, if we leave this as you have it here, i'd add a comment indicating why we're specifically choosing to `sleep` for now...